gasLimits.js

```diff
@@ -15,7 +15,8 @@ const gasLimitsConfig =
   "gasLimit_arc": arcForgeOrgGasLimit,
   /**
    * Gas limit sufficient (though not necessarily optimal) for all other transactions
-   * besides creating DAOs.  This is taken from the default that truffle puts in truffle.js.
+   * besides creating DAOs and instantiating a new GenesisProtocol.
+   * This is taken from the default that truffle puts in truffle.js.
```

Comment:
why did you choose to use truffle default values ? why do we care about truffle default value ?
how did truffle choose this value ?

Reply:
I think the code that uses this may not be required.  It is very old code that actually predates me, and has never, that I recall, been questioned.  I will test with it removed and report back.

BTW I am also looking into removing the other setting, `gasLimit_arc`, a task which is proving to be tricky, but with both of those settings gone we could completely remove some code including the dependency on the Arc test/constants.js file.